crates/bevy_ecs/Cargo.toml

```diff
@@ -21,6 +21,7 @@ bevy_utils = { path = "../bevy_utils", version = "0.8.0-dev" }
 bevy_ecs_macros = { path = "macros", version = "0.8.0-dev" }
 
 async-channel = "1.4"
+thread_local = "1.1.4"
```

Comment:
Is this already in our tree?

Reply:
No, but it is a nice crate that another PR (#4663) depends on as well.